### ISSUE TITLE
Can not use xmlchange or xmlquery with variables in env_batch.xml

### ISSUE THREAD

**Issue by mnlevy1981** [CESM-Development/cime#251]
env_batch.xml was not in the list of files that xmlchange or xmlquery would look in, so commands like

```
$ ./xmlchange JOB_WALLCLOCK_TIME=1:00
```

and

```
$ ./xmlquery JOB_WALLCLOCK_TIME
```

both failed. Pull request coming momentarily.

**Comment by jedwards4b** [comment]
​xmlchange already works but you need to use the old style
xmlchange -file env_batch.xml -id PROJECT -val WHATEVA ​

On Tue, Oct 20, 2015 at 6:22 PM, Michael Levy notifications@github.com
wrote:

> env_batch.xml was not in the list of files that xmlchange or xmlquery
> would look in, so commands like
> 
> $ ./xmlchange JOB_WALLCLOCK_TIME=1:00
> 
> and
> 
> $ ./xmlquery JOB_WALLCLOCK_TIME
> 
> both failed. Pull request coming momentarily.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/CESM-Development/cime/issues/251.

## 

Jim Edwards

CESM Software Engineer
National Center for Atmospheric Research
Boulder, CO

**Comment by mnlevy1981** [comment]
Is there a reason to require the -file flag for env_batch.xml but not any of the other files? (And thanks for pointing this option out, I didn't know the flag existed.)

**Comment by jedwards4b** [comment]
I guess that you've shown me that there is not.   Thanks

On Tue, Oct 20, 2015 at 7:43 PM, Michael Levy notifications@github.com
wrote:

> Is there a reason to require the -file flag for env_batch.xml but not any
> of the other files? (And thanks for pointing this option out, I didn't know
> the flag existed.)
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/CESM-Development/cime/issues/251#issuecomment-149754374
> .

## 

Jim Edwards

CESM Software Engineer
National Center for Atmospheric Research
Boulder, CO

**Comment by jedwards4b** [comment]
Fixed in #252
